source/index.ts

```diff
@@ -258,11 +258,11 @@ namespace is { // tslint:disable-line:no-namespace
 		'nodeValue'
 	];
 
-	export const domElement = (value: any): value is DomElement => object(value) as unknown && value.nodeType === NODE_TYPE_ELEMENT && string(value.nodeName) &&
-		!plainObject(value) && DOM_PROPERTIES_TO_CHECK.every(property => property in value);
+	export const domElement = (value: unknown): value is DomElement => object(value) && (value as DomElement).nodeType === NODE_TYPE_ELEMENT && string((value as DomElement).nodeName) &&
+		!plainObject(value) && DOM_PROPERTIES_TO_CHECK.every(property => property in (value as DomElement));
 
-	export const observable = (value: any) => Boolean(value && value[symbolObservable] && value === value[symbolObservable]());
-	export const nodeStream = (value: any): value is NodeStream => !nullOrUndefined(value) && isObject(value) as unknown && function_(value.pipe) && !observable(value);
+	export const observable = (value: unknown) => Boolean(value && (value as any)[symbolObservable] && value === (value as any)[symbolObservable]());
```

Comment:
Could we use `Object` instead of `any` here?

Reply:
We are lacking the index signature there on `Object`, `{}`, `object` etc.

I also can't wrap my head around what type to cast it to, attempting to type it out with a `symbolObservable` as index signature doesn't work, because types of index signatures have to be `string | number` thus it gives me:

```
[ts] Type 'symbol' cannot be used as an index type.
```

That's why I settled for any 🤷‍♂️👍

Reply:
Ah I see. Alright `any` is fine then.